events/wmt-terminology.md

```diff
@@ -0,0 +1,55 @@
+---
+parent: Events
+title: WMT Terminology 
+description: Terminology for understanding WMT results
+---
+
+## TrueSkill
+
+TrueSkill is an adaptative model of competitions originally developed by Microsoft Research for the Xbox Live online gaming community.
+[WMT](wmt.md) developed a variation of the TrueSkill model adapted to the human evaluation of machine translation output.
+During [WMT15](wmt15.md) and [WMT16](wmt16.md), the human ranking for each task was produced using TrueSkill in the following manner:
+
+First, 1,000 bootstrap-resampled runs are produced over all of the available data.
+A rank range for each system is then computed.
+To compute the rank range, the absolute rank of each system is collected in each fold and thrown out the top and bottom 2.5%.
```

Comment:
"and thrown out .the top and bottom 2.5%" is ungrammatical here

chartac naxadasutyun a eli